src/client/DAOClient.ts

```diff
@@ -353,6 +354,17 @@ class DAOClient implements zDAO {
       throw new Error(errorMessageForError('invalid-proposal-duration'));
     }
 
+    if (payload.transfer) {
+      if (payload.choices && payload.choices.length !== 2) {
```

Comment:
Consider simplifying this logic a bit by first checking `if (!payload.choices)`''

```typescript
if (payload.transfer) {
  if (!payload.choices) {
      payload.choices = DEFAULT_CHOICES;
  } else if (payload.choices.length !== 2) // we know payload.choices is valid here, no need for duplicative check
      throw new Error(...);
  }
}

Reply:
Thanks for your review, that makes sense

Reply:
I am thinking if client didn't pass choices, then use `DEFAULT_CHOICES` by default.